src/view/UserInputDialog.py

```diff
@@ -26,18 +28,25 @@ def __init__(self, input_fields: list[str], button_name: str, window_title: str,
         if numerical_input_fields is None:
             numerical_input_fields = []
 
-        super().__init__(parent)
+        super().__init__(parent, QtCore.Qt.WindowSystemMenuHint | QtCore.Qt.WindowTitleHint)
```

Comment:
Adding these weirdly removes the ability to close the window with the 'X'-button. It is grayed out.
Do you have the same problem or is this OS-dependant?

Reply:
Nope I don't have the Problem. I just googled a Solution to get rid of the questionmark and that came up. But I couldn't test it because I don't have a question mark :/ So I will just remove them again.